app/src/components/GrantList.vue

```diff
@@ -1,85 +1,44 @@
-        >
-          <div class="w-full flex items-center justify-between p-6 space-x-6 hover:border">
-            <div class="flex-1 truncate text-left">
-              <div class="flex items-center space-x-3">
-                <h3 class="text-gray-900 text-sm font-medium truncate">
-                  Grant Name: {{ grantMetadata[grant.metaPtr].name }}
-                </h3>
-              </div>
-              <p class="mt-1 text-gray-500 text-sm truncate">{{ grantMetadata[grant.metaPtr].description }}</p>
-            </div>
-          </div>
-          <div>
-            <div class="pl-6 p-2 -mt-px flex divide-x divide-gray-400 divide-opacity-30">
-              <div class="w-0 flex-1 flex">
-                <div class="flex-1 truncate text-left">
-                  <p class="mt-1 text-gray-500 text-sm truncate">Owner</p>
-                  <div class="flex items-center space-x-3">
-                    <h3 class="text-gray-900 text-sm font-medium">{{ formatAddress(grant.owner) }}</h3>
-                  </div>
-                </div>
-              </div>
-              <div class="pl-6 -ml-px w-0 flex-1 flex">
-                <div class="w-0 flex-1 flex">
-                  <div class="flex-1 truncate text-left">
-                    <p class="mt-1 text-gray-500 text-sm truncate">Payee</p>
-                    <div class="flex items-center space-x-3">
-                      <h3 class="text-gray-900 text-sm font-medium">{{ formatAddress(grant.payee) }}</h3>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="divide-x divide-gray-400 divide-opacity-30 flex justify-center">
-          <button v-if="isInCart(grant.id)" @click="removeFromCart(grant.id)" class="my-2 btn btn-primary">
-            Remove from Cart
-          </button>
-          <button v-else @click="addToCart(grant.id)" class="my-2 btn btn-primary">Add to Cart</button>
-        </div>
+    <!-- ToDo. Loading... message-->
+    <ul class="text-left grid grid-cols-1 gap-10 py-10 sm:grid-cols-2">
+      <li v-for="grant in grants" :key="grant.id.toString()">
+        <!-- ToDo: Raised data -->
+        <!-- ToDo: Grant image -->
+        <GrantCard
+          :id="grant.id"
+          :name="grantMetadata[grant.metaPtr].name ?? ''"
+          :ownerAddress="grant.owner"
+          raised="100"
+          imgurl="https://s3-alpha-sig.figma.com/img/901c/c3ae/ac690f9a43a7634d9d198cf9057c4454?Expires=1632096000&Signature=PnDBcqyesPYCxYzyRTGn-imh2ktN8ImD4GIrtJKoBrpooTX6Uqae5ogcpWdMpn6CyZnfDAnGuA2XYokhIpLTbLTriqd5Ifiiv2zbVHHQlUwQQ8fk4k9gpAj17CTBJGS0Ldqv6pn9MvSTzXKaLa~FjSiX2AAeWgWz1Vy9OJ6KFahsmhfLXlGEmBDr1Dx016AMvWH5DhKRLY4XbpfJz5zJy6zEHbV7GZ6uxop4d-pIk9KvTPt3kjJEsmKRdVxxy8XC8zN2cIFMEQEWJDRe64AekEl5ABzlTN~9o8NYS6~q5ObH3BlwQuvUhPLNvw0Atjwc9DX3yWuo-cAOZISl5sboOg__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA"
```

Comment:
This should be pulled from the GrantMetadata, looks like it's still hardcoded

Reply:
The image URL is not coming (yet, I guess) in the GrantMetadata. I left a couple of ToDo comments above the component.

Reply:
Our `GrantMetadata` type has an optional `logoURI` property, and we do resolve grant metadata currently, so we should be able to use the actual grant image now right? And we should just fallback to `placeholder_grant.svg` if the grant metadata has no `logoURI`

Reply:
You're completely right. I was just checking the actual test metadata I was getting, but didn't notice the `GrantMetadata.logoURI` property.

Fixed in https://github.com/dcgtc/dgrants/pull/184/commits/7299102e963653726b6b0e9721df09fa71267c6f

Thanks Matt!

Reply:
LGTM, thanks!